mimic/rest/maas_api.py

```diff
@@ -1299,6 +1304,31 @@ def multiplot(self, request, tenant_id):
         self._audit('rollups', request, tenant_id, status, content)
         return json.dumps({'metrics': metrics_replydata})
 
+    @app.route('/v1.0/<string:tenant_id>/views/latest_alarm_states', methods=['GET'])
+    def latest_alarm_states(self, request, tenant_id):
+        """
+        Gets entities grouped with their latest alarm states.
```

Comment:
There are docs for the `/views/overview` API, but not this one. The API for creating alarm states is part of the Mimic control plane. If you have a preferred way of writing up the APIs that we create for Mimic, let me know and I'll send a follow-up PR.